bigtable/scheduled-backups/src/test/java/com/example/cloud/bigtable/scheduledbackups/CreateBackupTest.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloud.bigtable.scheduledbackups;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.gax.grpc.GrpcStatusCode;
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.api.gax.longrunning.OperationFutures;
+import com.google.api.gax.longrunning.OperationSnapshot;
+import com.google.api.gax.rpc.OperationCallable;
+import com.google.api.gax.rpc.UnaryCallable;
+import com.google.api.gax.rpc.testing.FakeOperationSnapshot;
+import com.google.bigtable.admin.v2.CreateBackupMetadata;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.admin.v2.internal.NameUtil;
+import com.google.cloud.bigtable.admin.v2.models.Backup;
+import com.google.cloud.bigtable.admin.v2.models.CreateBackupRequest;
+import com.google.cloud.bigtable.admin.v2.stub.EnhancedBigtableTableAdminStub;
+import com.google.longrunning.Operation;
+import com.google.protobuf.Timestamp;
+import com.google.protobuf.util.Timestamps;
+import io.grpc.Status.Code;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.threeten.bp.Instant;
+
+
+@RunWith(JUnit4.class)
+public final class CreateBackupTest {
+  @Rule public final MockitoRule mockitoRule = MockitoJUnit.rule();
```

Comment:
There should also be an integration test for this - you should be able to set them up similarly to how other bigtable ITs are set up in this directory

Reply:
Thanks for noting! We are planning to add integration test for this (i.e. following the steps in the script and check a backup is created), is there a good way to do it here?

Reply:
I'm not 100% sure about how to do an end to end test with gcloud commands and scrips - @lesv and @shubha-rajan for thoughts. At the very least, I think we should instead add an integration test for https://github.com/GoogleCloudPlatform/java-docs-samples/pull/4590/files#diff-5741fc3e72b818817f6149514616f5384e53f589c99a38a98b6ebc9ae7034d4e rather than a unit test with mocks. Since this is only testing the client library usage, I don't think this is necessary as that's already tested in the client library. Let me know if you want to chat about this further!

Reply:
We want tests to accept the pr

Reply:
Integration tests see sample format doc in the root.

Reply:
Typically when we have gcloud commands in the READMEs, we don't test them (whether we should, is another question imo). I think since the shell scripts are just taking deploy commands that would typically be in the README, and putting them in a script for convenience, the same rules apply. 

The cloud function should definitely be tested.